src/spring-cloud/azext_spring_cloud/commands.py

```diff
@@ -101,19 +103,22 @@ def load_command_table(self, _):
         g.custom_show_command('show', 'spring_cloud_get', table_transformer=transform_spring_cloud_table_output)
 
     with self.command_group('spring-cloud test-endpoint', client_factory=cf_spring_cloud_20220101preview,
+                            deprecate_info=self.deprecate(target='spring-cloud', redirect='spring'),
```

Comment:
It is recommended to write the redirect command signature in detail to avoid confusing users
```suggestion
    with self.command_group('spring-cloud test-endpoint', client_factory=cf_spring_cloud_20220101preview,
                            deprecate_info=self.deprecate(redirect='spring test-endpoint'),
```

Reply:
updated